services/GoogleSitemapService.cfc

```diff
@@ -176,23 +196,48 @@ component {
 		return page.sitemap_priority;
 	}
 
-	private function _addChildPages( required array haveAccessPages, required array childPages, string parentSearchEngineAccess, string parentAccessRestriction, string parentSitemapPriority ) {
+	private string function _getSitemapChangeFreqForPage( required string pageId ) {
+		var page = _getSiteTreeService().getPage( id=arguments.pageId, selectFields=[ "id", "parent_page", "sitemap_change_freq" ] );
+
+		if ( !page.recordCount ) {
+			return "always";
+		}
+		if ( !Len( Trim( page.sitemap_change_freq ?: "" ) ) || page.sitemap_change_freq == "inherit"   ) {
+			if ( Len( Trim( page.parent_page ) ) ) {
+				return _getSitemapPriorityForPage( page.parent_page );
```

Comment:
I think this should be `_getSitemapChangeFreqForPage()`?

Reply:
Hi Dom,

Updated the error function call within _getSitemapChangeFreqForPage()
did a test and sitemap is generated correctly.

Thank you
Nelson